envoy/datadog_checks/envoy/envoy.py

```diff
@@ -13,6 +13,7 @@
 
 
 class Envoy(AgentCheck):
+    HTTP_CONFIG_REMAPPER = {'verify_ssl': {'name': 'tls_verify'}}
     SERVICE_CHECK_NAME = 'envoy.can_connect'
 
     def __init__(self, name, init_config, agentConfig, instances=None):
```

Comment:
And modify elsewhere too

```suggestion
    def __init__(self, name, init_config, instances):
```

Reply:
Dunno what's up with that leading space suggestion... GitHub 🙂

Reply:
Modify elsewhere what?

Reply:
To reflect the change in signature

Reply:
ohhh, I ignored everythin other than the whitespace 🤦‍♂